examples/hn.svelte.dev/src/routes/__error.svelte

```diff
@@ -1,4 +1,5 @@
 <script context="module">
+	/** @type {import('@sveltejs/kit').ErrorLoad} */
```

Comment:
this is kind of a big PR. what do you think about sending the changes to `hn.svelte.dev` in a separate PR to break things up a bit? we could merge that change more quickly